### PR TITLE
[FIX] web: in many2many in a wizard the last change is not saved.

### PR DESCRIPTION
**Pull request by Gorash**
Need to use ajax instead session because we can't add the required if the depends are not clean.

opw-24068

### CODE REVIEW

File: addons/web/static/src/js/views/form_relational_widgets.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ var data = require('web.data');
 var Dialog = require('web.Dialog');
 var common = require('web.form_common');
 var ListView = require('web.ListView');
+require('web.ListEditor'); // require because list editable because add the start_edition methods
```
Comment (ged-odoo) [inline addons/web/static/src/js/views/form_relational_widgets.js]: your comment dont not unmake sense...

seriously, I have no idea what you are trying to say

File: addons/web/static/src/js/framework/session.js [hunk 2]
```diff
@@ -141,9 +141,12 @@ var Session = core.Class.extend(mixins.EventDispatcherMixin, {
         }
         var def = this._groups_def[group];
         if (!def) {
-            var Model = window.openerp.web.Model;
-            var Users = new Model('res.users');
-            def = this._groups_def[group] = Users.call('has_group', [group]);
+            def = this._groups_def[group] = this.rpc('/web/dataset/call_kw/res.users/has_group', {
```
Comment (ged-odoo) [inline addons/web/static/src/js/framework/session.js]: one less call to openerp! yay :)

Comment (ged-odoo) [inline addons/web/static/src/js/framework/session.js]: hmm, how does this work? you explicitely use 'base.group_no_one' and not the variable group?

File: addons/web/static/src/js/views/form_relational_widgets.js (hunk 1 above)
Comment (Gorash) [inline addons/web/static/src/js/views/form_relational_widgets.js]: lol je ne sais même plus ce qui est écrit, on dirait la fusion de plusieurs phrases ^^

Il faut juste le require pour être sûr que l'include soit fait

**Comment by ged-odoo** [comment]
merged (with slightly different message), in 10.0: https://github.com/odoo/odoo/commit/3378f9c771910fb61a13a452a8f8a1fb577b9c23

**Pull request closed without merging**